src/docs/product/issues/issue-details/performance-issues/regressions.mdx

```diff
@@ -0,0 +1,39 @@
+---
+title: "Function Regression"
+sidebar_order: 40
+description: "Learn more about Function Regression Issue and how to diagnose and fix them."
+---
+
```

Comment:
I believe we should include the EA note while this is in EA.

```suggestion

<Include name="feature-available-for-user-group-early-adopter.mdx" />

```

Reply:
And can we add a link to this page to the list on the Early Adopter Features page: src/docs/product/accounts/early-adopter-features/index.mdx ?